packages/database/src/core/util/libs/parser.ts

```diff
@@ -72,7 +72,7 @@ export const parseRepoInfo = function(
   const parsedUrl = parseDatabaseURL(dataURL),
     namespace = parsedUrl.namespace;
 
-  if (parsedUrl.domain === 'firebase') {
+  if (parsedUrl.domain === 'firebase.com') {
```

Comment:
Isn't this a breaking change for users that use `firebase.com?ns=foo`?

Reply:
Never mind. This is only breaking for `firebase/ns?foo` which shouldn't affect anyone.

Reply:
I believe in the past. `*.firebase.*` will hit this.

My guess is that RTDB was served in `firebase.com` instead of `firebaseio.com` in the past.

Added a test for this.